zokrates_core/src/proof_system/bn128/utils/parser.rs

```diff
@@ -0,0 +1,45 @@
+use std::collections::HashMap;
+
+pub struct KeyValueParser;
```

Comment:
I'm wondering if we can get this done with the csv crate (we already use to (de)serialize the witness) by setting the delimiter here https://docs.rs/csv/1.1.3/csv/struct.ReaderBuilder.html#method.delimiter